relay-server/src/actors/outcome.rs

```diff
@@ -326,6 +326,9 @@ pub enum DiscardReason {
     /// (Relay) The profile is parseable but semantically invalid. This could happen if
     /// profiles lack sufficient samples.
     InvalidProfile,
+
+    // (Relay) We failed to parse the replay so we discard it.
+    InvalidReplay,
```

Comment:
should we do `InvalidReplayEvent`? just to split ambiguity between event / segment

Reply:
👍